src/plugins/simulator/fs9/simulatorfs9.cpp

```diff
@@ -418,9 +433,9 @@ namespace BlackSimPlugin::Fs9
         {
             const auto dontOverwrite = [](const CFrequency &from, const CFrequency &to) {
```

Comment:
As the ``to`` parameter is unused now, it should be removed. This will also fix the compiler warning (unreferenced formal parameter).

Reply:
done